app/views/records/edit_fields/_subject.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= f.input key,
```

Comment:
I'm unclear on why this partial isn't included from Hyrax 2.0 already. The other edit field partials (e.g. [description](https://github.com/samvera/hyrax/blob/v2.0.0/app/views/records/edit_fields/_description.html.erb)) seem to work out of the box, right?

Does adding this file change application behavior immediately in some testable way? Or should this commit be squashed into another with code that uses it?